docs/bfb.md

```diff
@@ -97,22 +97,52 @@ MQTT Topic: selfupdate/desiredstatefeedback
 }
 ```
 
+## Command for single self-update action/step
+This message contains information about SUA action - what must be done as a single step - download, update, activate, cleanup.
+MQTT Topic: selfupdate/desiredstate/command
+```
+{
+    "activityId": "random-uuid-as-string",
+    "timestamp": 123456789,
+    "payload": {
+        "baseline": "BASELINE NAME",
+        "command": "command_type"
+    }
+}
+```
+
 ## Description of statuses
 Following combination of payload_status and action_status are possible:
-| payload_status | action_status | Description |
-|----------------|---------------|-------------|
-| IDENTIFYING | | SUA has received request for update and evaluating it |
-| IDENTIFIED | | SUA has received request for update and will try to perform an update |
-| RUNNING | DOWNLOADING | Downloading an image |
-| RUNNING | DOWNLOAD_SUCCESS | Image is downloaded without errors |
-| RUNNING | INSTALLING | Installing an image |
-| INCOMPLETE | UPDATE_FAILURE | One of these has failed: download, update or self-update was rejected. Message will contain detailed description what has happened |
-| COMPLETED | UPDATE_SUCCESS | Self-update succeeded |
+| payload_status        | action_status    | Description |
+|-----------------------|------------------|-------------|
+| IDENTIFYING           |                  | SUA has received request for update and evaluating it |
+| IDENTIFICATION_FAILED |                  | SUA has received request for update and unable to perform self-update |
+| IDENTIFIED            | IDENTIFIED       | SUA has received request for update and will try to perform an update |
+| DOWNLOADING           | DOWNLOADING      | Downloading an image |
+| DOWNLOAD_SUCCESS      | DOWNLOAD_SUCCESS | Image is downloaded without errors |
+| DOWNLOAD_FAILURE      | DOWNLOAD_FAILURE | Image was not downloaded |
+| UPDATING              | UPDATING         | Installing an image |
+| UPDATE_SUCCESS        | UPDATING         | Image installed |
+| UPDATE_FAILURE        | UPDATE_FAILURE   | Image was not installed due to error |
+| ACTIVATING            | UPDATING         | SUA is activating partition with new image |
+| ACTIVATION_SUCCESS    | UPDATED          | SUA has activated partition with new image |
```

Comment:
"...activate/ing/ed _the_ partition..."  (w/o "the" it is "an" partition with new image.)   - in 3 descr.